### PR TITLE
Suggest using Quarkiverse Hub to publish extension

### PR DESCRIPTION
**Pull request by gastaldi**
Also notify Quarkiverse maintainers about new kind/extension-proposal tickets

### CODE REVIEW

File: docs/src/main/asciidoc/building-my-first-extension.adoc [hunk 1]
```diff
@@ -805,9 +805,15 @@ If the extension solves a general problem, it is very handy for Quarkus users to
 But this comes with some responsibility for you, keeping it up to date with Quarkus minor releases (every month or so at the moment).
 When in doubt, have a conversation with the community in the https://groups.google.com/forum/#!forum/quarkus-dev[Quarkus Google Group]. We can make a collective decision.
 
-[NOTE]
-As for now, the process to propose a new extension is not defined yet.
-Your best chance is to present your extension on the https://groups.google.com/forum/#!forum/quarkus-dev[Quarkus Google Group] and wait for an official invitation to join the Quarkus Platform.
+If you want to contribute with a new extension, we strongly suggest you to host it in the Quarkiverse Hub.
```
Comment (FroMage) [inline docs/src/main/asciidoc/building-my-first-extension.adoc:808]:
```suggestion
If you want to contribute a new extension, we strongly suggest you to host it in the Quarkiverse Hub.
```

**Review by FroMage — APPROVED** [review]

File: docs/src/main/asciidoc/building-my-first-extension.adoc (hunk 1 above)
Comment (gastaldi) [reply docs/src/main/asciidoc/building-my-first-extension.adoc:808]: Actually the correct is `contribute to`: https://dictionary.cambridge.org/us/dictionary/english/contribute-to-sth

Comment (FroMage) [reply docs/src/main/asciidoc/building-my-first-extension.adoc]: Well it depends what you mean:

- "contribute to a new extension" means the extension already exists and you want to contribute to it
- "contribute a new extension" means adding a new extension as a contribution

Comment (gastaldi) [reply docs/src/main/asciidoc/building-my-first-extension.adoc]: Ah that makes sense, let me fix that

File: docs/src/main/asciidoc/building-my-first-extension.adoc [hunk 2]
```diff
@@ -805,9 +805,15 @@ If the extension solves a general problem, it is very handy for Quarkus users to
...
-[NOTE]
-As for now, the process to propose a new extension is not defined yet.
-Your best chance is to present your extension on the https://groups.google.com/forum/#!forum/quarkus-dev[Quarkus Google Group] and wait for an official invitation to join the Quarkus Platform.
+If you want to contribute a new extension, we strongly suggest you to host it in the Quarkiverse Hub.
+
+*Quarkiverse Hub*
+
+link:https://github.com/quarkiverse[Quarkiverse Hub] is the GitHub organization that provides repository hosting (including build, CI and release publishing setup) for Quarkus extension projects contributed by the community.
+You can get started by creating an link:https://github.com/quarkusio/quarkus/issues/new/choose[Extension Request] issue (check first if one wasn't already submitted link:https://github.com/quarkusio/quarkus/labels/kind%2Fextension-proposal[here]) and asking to lead it.
+We'll take care provisioning a new repository and set it up to our tooling.
```
Comment (gsmet) [inline docs/src/main/asciidoc/building-my-first-extension.adoc:814]:
```suggestion
We'll take care of provisioning a new repository and set it up to our tooling.
```

File: docs/src/main/asciidoc/building-my-first-extension.adoc [hunk 3]
```diff
@@ -805,9 +805,15 @@ If the extension solves a general problem, it is very handy for Quarkus users to
...
+
```
Comment (gsmet) [inline docs/src/main/asciidoc/building-my-first-extension.adoc:815]: I wonder if we should somehow say that we give the key of the repository to the requester?

Comment (gastaldi) [reply docs/src/main/asciidoc/building-my-first-extension.adoc:815]: What key are you referring to? The repository owner has limited access to the repository settings AFAIK

Comment (gsmet) [reply docs/src/main/asciidoc/building-my-first-extension.adoc:815]: Well it's more than people are totally independent to manage their project and releases. I think it's an important factor when taking the decision of Quarkiverse vs my own repo.

But maybe it's not worth it. I'll let you judge of that. Feel free to merge if you don't think it's necessary.

Comment (gastaldi) [reply docs/src/main/asciidoc/building-my-first-extension.adoc:822]: I see, that's a good idea and a nice motivation to use Quarkiverse. I reworded it a bit, let me know how that looks now

File: docs/src/main/asciidoc/building-my-first-extension.adoc [hunk 4]
```diff
@@ -805,9 +805,22 @@ If the extension solves a general problem, it is very handy for Quarkus users to
...
+*Quarkiverse Hub*
+
+link:https://github.com/quarkiverse[Quarkiverse Hub] is the GitHub organization that provides repository hosting (including build, CI and release publishing setup) for Quarkus extension projects contributed by the community.
+
+You can get started by creating an link:https://github.com/quarkusio/quarkus/issues/new/choose[Extension Request] issue (check first if one wasn't already submitted link:https://github.com/quarkusio/quarkus/labels/kind%2Fextension-proposal[here]) and asking to lead it.
+
+We'll take care of provisioning a new repository and set it up to:
+
+- Be supported by our tooling;
+- Publish the documentation you produce for your extension in the Quarkiverse website;
```
Comment (gsmet) [inline docs/src/main/asciidoc/building-my-first-extension.adoc:819]:
```suggestion
- Publish the documentation you produce for your extension to the Quarkiverse website;
```

File: docs/src/main/asciidoc/building-my-first-extension.adoc [hunk 5]
```diff
@@ -805,9 +805,22 @@ If the extension solves a general problem, it is very handy for Quarkus users to
...
+- Configure your extension to use the link:https://github.com/quarkusio/quarkus-ecosystem-ci#quarkus-ecosystem-ci[Quarkus Ecosystem CI] to build against the latest Quarkus Core changes;
+- Give you the freedom to manage the project and releases to Maven central as you like.
```
Comment (gsmet) [inline docs/src/main/asciidoc/building-my-first-extension.adoc:821]:
```suggestion
- Give you the freedom to manage the project and release to Maven Central as you like.
```

**Review by gsmet — CHANGES_REQUESTED** [review]
Added a few minor suggestions. Other than that, it looks very good!

**Review by gsmet — APPROVED** [review]
I squashed the last commit. Let's get this merged once CI is green.

**Pull request merged**